test/config.spec.coffee

```diff
@@ -60,8 +60,6 @@ describe 'AWS.Config', ->
       expect(configure(maxRetries: 2).maxRetries).to.equal(2)
 
   describe 'retryDelayOptions', ->
-    it 'defaults to "base: 100"', ->
```

Comment:
Optional: you might want to add a test to service.spec.coffee verifying that a normal client still gets a retry base of 100 set.